src/api/app/controllers/webui/package_controller.rb

```diff
@@ -390,8 +393,9 @@ def set_file_details
       end
     rescue ActiveXML::Transport::Error => e
       # TODO crudest hack ever!
-      if e.summary == 'service in progress'
+      if e.summary == 'service in progress' && @expand == 1
         @expand = 0
+        @service_running = true
```

Comment:
Why is this important? I don't see this variable being used inside the view nor this controller (at least not after line 398).